doc/adoc/user_guide.adoc

```diff
@@ -210,6 +210,12 @@ a regular `reboot` can be requested by setting `soft_reboot: false`.
 [listing]
 soft_reboot: true|false
 
+Enable verbosity for zypper migration::
+If enabled, it will run the zypper migration plugin with increased verbosity.
+
+[listing]
+verbosity_migration: true|false
```

Comment:
s/verbosity/verbose/

Reply:
Ack, updated doc. schema, code and tests